src/components/common/Extensions/UserHub/partials/StakesTab/partials/StakesList.css

```diff
@@ -0,0 +1,38 @@
+@value query700 from "~styles/queries.css";
+
+.stakesListContainer {
+  padding-right: 5px;
+  height: 60vh;
+  max-height: 356px;
+  width: 100%;
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+/* The scrollbar itself */
+.stakesListContainer::-webkit-scrollbar {
```

Comment:
There's also https://www.npmjs.com/package/tailwind-scrollbar. Maybe make a note for later.